content/docs/dvclive/api-reference/log.md

```diff
@@ -0,0 +1,53 @@
+# dvclive.log()
+
+Generates _plot metrics_ by saving the given `name`: `val` pair to a `.tsv`
+file.
+
+```py
+ def log(name: str, val: Union[int, float], step: int = None) -> None:
```

Comment:
Another really minor thought: In the dvc api docs, we omit the return type. Not sure if there's a style preference here, just a thought on consistency. cc @skshetry?

Reply:
I saw the return type being used in get_url and followed that:

https://dvc.org/doc/api-reference/get_url

I should have probably checked the other pages for consistency